examples/prerecorded_from_file/main.rs

```diff
@@ -0,0 +1,26 @@
+use deepgram::{BufferSource, Deepgram, DeepgramError, Language, OptionsBuilder};
+use tokio::fs::File;
+
+const DEEPGRAM_API_KEY: &str = "YOUR_SECRET";
+const PATH_TO_FILE: &str = "examples/prerecorded_from_file/Bueller-Life-moves-pretty-fast.wav";
+
+#[tokio::main]
+async fn main() -> Result<(), DeepgramError> {
+    let dg_client = Deepgram::new(DEEPGRAM_API_KEY);
+
+    let file = File::open(PATH_TO_FILE).await.unwrap();
+
+    let source = BufferSource {
+        buffer: file,
+        mimetype: Some("audio/wav"),
+    };
+
+    let options = OptionsBuilder::new()
+        .punctuate(true)
+        .language(Language::en_US);
+
+    let response = dg_client.prerecorded_request(source, &options).await?;
```

Comment:
what do you like better API-wise between what you have here and
```
dg_client.prerecorded()
    .source(source)
    .punctuate(true)
    .language(Language::en_US)
    .send()
    .await?;
```

Reply:
I can see both being good right now, having a separate `Options` struct like what you have means that you can set that up once and then use it for multiple requests, which I really like. Putting it all on `PrerecordedRequestBuilder` means one less thing to import, which I thought I liked, but now that I'm looking at it I'm not sure

Reply:
The way I have it currently is more similar to our other SDKs (the Python one, at least). I think I like your suggestion more, since it's a lot more similar to how Reqwest works, which Rust programmers might be more familiar with.

We'd probably want something more like the following, since the source isn't optional:

```rust
dg_client.prerecorded(source)
    .punctuate(true)
    .language(Language::en_US)
    .send()
    .await?;
```